lms/static/certificates/sass/_layouts.scss

```diff
@@ -0,0 +1,324 @@
+// ------------------------------
+// // Open edX Certificates: Layouts
+
+// About: styling for generic layouts and responsive support.
+
+// #BASE
+// #ACCOMPLISHMENT
+
+// ------------------------------
+// #BASE
+// ------------------------------
+%layout-wrapper {
+    margin-bottom: spacing-vertical(base);
+    padding: 0 (gutter()*2);
+
+    @include susy-breakpoint($bp-medium, $susy) {
+        padding: 0 gutter();
+    }
+}
+
+%layout {
+    @include container();
+}
+
+// app header
+.wrapper-header {
+    @extend %layout-wrapper;
+    padding-top: spacing-vertical(small);
+    padding-bottom: spacing-vertical(small);
+}
+
+.header-app {
+    @extend %layout;
+
+    .logo {
+        width: span(2);
+        margin: 0 auto;
+    }
+}
+
+.header-app-title {
+    margin: 0;
+    text-align: center;
+}
+
+// banner
+.wrapper-banner {
+    @extend %layout-wrapper;
+    padding-top: spacing-vertical(small);
+    padding-bottom: spacing-vertical(small);
+
+    .banner {
+        @extend %layout;
+    }
+}
+
+// message to user-centric banner
+.wrapper-banner-user {
+    margin-top: -(spacing-vertical(base)); // abutt the global header, plz
+}
+
+.banner-user {
+    text-align: center;
+
+    @include susy-breakpoint($bp-large, $susy) {
+        text-align: left;
+    }
+
+    .wrapper-copy-and-actions {
+        @include container();
+
+        .message-copy {
+            margin-bottom: spacing-vertical(small);
+
+            @include susy-breakpoint($bp-large, $susy) {
+                @include span(8 first);
```

Comment:
NOTE: new susy grid-management syntax - ``span(*)`` is a method for laying out columns. See http://susydocs.oddbird.net/en/latest/shorthand/#span.